index.js

```diff
@@ -129,6 +138,11 @@ class Ky {
 
 				const response = await this._response;
 
+				for (const hook of this._hooks.afterResponse) {
+					// eslint-disable-next-line no-await-in-loop
+					await hook(response.clone());
```

Comment:
Isn't the point of having a `afterResponse` hook that the hook can (optionally) return a new Response object?